src/components/UI/Link/styles.ts

```diff
@@ -1,12 +1,15 @@
 import { theme } from '@/theme';
 
-export const linkStyles = {
-  root: {
-    backgroundColor: 'transparent',
-  },
-  label: (hovered: boolean) => ({
-    textDecoration: 'none',
-    fontWeight: '500',
-    color: hovered ? theme.colors.cyan[4] : theme.white,
-  }),
+export const linkStyles = (isDark: boolean) => {
+  const colortext = isDark ? theme.white : theme.colors.gray[7];

```

Comment:
Same question in here as well, i dont really like the idea of us converting this entire file into a function